source/_docs/hsts.md

```diff
@@ -51,3 +71,31 @@ Once installed and configured, the following header will be sent in responses:
 ```http
 strict-transport-security: max-age=15552000
 ```
+</div>
+
+</div>
+
+
+## Configure Attributes
```

Comment:
Suggest adding instructions to this section for how to configure attributes. If step by step can't be provided, maybe some general advice for how to do this?

Reply:
Good idea, I'll see if @peter-pantheon can look into this during his next doc block.

Reply:
@rachelwhitton @alexfornuto 

User's can inject the header via a (configurable) WordPress plugin, (configurable)  Drupal module, code or by hard coding the header into a template file. So I think giving general advisement on configuration options is acceptable here.  

I see there is a header called "Configure Atrributes" and info below it and that looks good to me. What more specifics would be desired here? 

There are some details here that may require further offline discussion too,